packages/v4/src/content/design-guidelines/components/tabs/tabs.md

```diff
@@ -2,4 +2,64 @@
 id: Tabs
 section: components
 ---
-A **tab** component creates a set of tabs to organize content on a page. It must always be used together with a tab content component.
+**Tabs** allow users to navigate between views within the same page or context.
+
+## Usage
+
+PatternFly offers a few types of tabs:
+
+- [Default tabs](#default-tabs) (recommended)
+- [Boxed tabs](boxed-tabs)
+- [Vertical tabs](#vertical-tabs)
+
+You can use as many tabs as you want in a page, although it is recommended you keep the total number of tabs to a minimum for a better user experience.If you have more tabs than fit on the page, you may use a [tab overflow](/components/tabs#default-overflow) which would allow a user to horizontal scroll through your tabs. The uses of both default and boxed stylings are flexible, so use either according to your needs and preferences. 
+
+### Default tabs
+
+Default tabs do not have any borders, other than the bottom line to distinguish between a selected tab, a hovered over tab, and a non-active tab. These tabs are most commonly used as top page header tabs or nested within components.
+
+<img src="./img/default-tabs.png" alt="Default tabs ex" width="1500"/>
+
+Default tabs can be formatted as standard or filled.  Filled tabs are placed side by side and span the whole width of the page. They can be used in screens with narrower widths, such as a drawer within a [primary detail view](/demos/primary-detail). When in doubt, stick with the standard styled tabs. 
```

Comment:
The use of "Filled styled" was a little confusing. The tabs themselves are not filled. It's more of a question of whether the tabs should stretch to fit to the container width or not. Maybe "standard" or "fit to container" would be clearer?

Reply:
The reason I used "filled" was because that's what it is called in the code https://www.patternfly.org/v4/components/tabs#filled. I can update the description here to be clearer though!